docs/functions-runtime-kubernetes.md

```diff
@@ -65,6 +65,58 @@ If the token you use to deploy the function has an expiration date, you may need
 
 :::
 
+### Enable Kubernetes Service Account Token Projection for Function Pod Authentication
+
+The `KubernetesServiceAccountTokenAuthProvider` uses [Service Account Token Volume Projections](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#serviceaccount-token-volume-projection) to mount a token into the function's pod. The function worker and broker can verify this token using OpenID Connect. The primary benefit of this integration is that tokens have a short time to live, are managed by Kubernetes, and do not inherit the permission used to create the function.
+
+:::note
+This feature requires that the broker and the function worker are configured to use the `AuthenticationProviderOpenID`. Documentation to enable this provider can be found [here](security-openid-connect.md).
+:::
+
+Here is an example configuration for the function worker to utilize this feature:
+
+```yaml
+functionAuthProviderClassName: "org.apache.pulsar.functions.auth.KubernetesServiceAccountTokenAuthProvider"
+kubernetesContainerFactory:
+  kubernetesFunctionAuthProviderConfig:
+    # Required
+    serviceAccountTokenExpirationSeconds: "600"
+    serviceAccountTokenAudience: "the-required-audience"
+    # Optional
+    brokerClientTrustCertsSecretName: "my-secret-pulsar-broker-client-trust-certs"
+```
+
+The function pod deploys with the default Kubernetes Service Account for the target namespace. Because the service account name maps to the `sub` claim on the JWT projected into the pod's filesystem, all pods with the same service account will have the same permission within Pulsar. There is ongoing work to improve this integration.
```

Comment:
```suggestion
The function pod deploys with the default Kubernetes Service Account for the target namespace. Because the service account name maps to the `sub` claim on the JWT projected into the pod's filesystem, all pods with the same service account will have the same permission within Pulsar. There is ongoing work to improve this integration.
```

Is Kubernetes Service Account a proper noun?